addon/ng2/tasks/npm-install.js

```diff
@@ -0,0 +1,17 @@
+/* jshint node: true, esversion: 6 */
```

Comment:
`npm link npm` caused a lot of problems with testing. See https://github.com/angular/angular-cli/pull/198